Tools-Override/tests.targets

```diff
@@ -198,6 +198,13 @@
       Overwrite="true"
       Encoding="Ascii" />
 
+      <!--======================================================
+                      Section for netfx test runs
+            ====================================================== -->
+    <ItemGroup Condition="'$(BuildingNETFxVertical)' == 'true'" >
+      <TestCommandLines Include="set DEVPATH=%RUNTIME_PATH%" />
```

Comment:
Actually @tarekgh is right. SETLOCAL is already being called at the begining of the batch file.